pennylane/devices/_qubit_device.py

```diff
@@ -317,13 +316,12 @@ def execute(self, circuit, **kwargs):
             shots_from_dev = self._shots if not self.shot_vector else self._raw_shot_sequence
             tape_resources = circuit.specs["resources"]
 
-            resources = Resources(  # temporary until shots get updated on tape !
-                tape_resources.num_wires,
-                tape_resources.num_gates,
-                tape_resources.gate_types,
-                tape_resources.gate_sizes,
-                tape_resources.depth,
-                Shots(shots_from_dev),
+            resources = SpecsResources(  # temporary until shots get updated on tape !
```

Comment:
```suggestion
            resources = SpecsResources(
```

Should I remove this comment? It seems to have been left here for a very long time